pkg/client/projectrequests.go

```diff
@@ -32,3 +37,10 @@ func (c *newProjectRequestsStruct) Create(p *projectapi.ProjectRequest) (result
 	err = c.r.Post().Resource("projectrequests").Body(p).Do().Into(result)
 	return
 }
+
+// List returns a status object indicating that a user can call the Create or an error indicating why not
+func (c *newProjectRequestsStruct) List(label labels.Selector, field fields.Selector) (result *kapi.Status, err error) {
```

Comment:
nothing wrong with method, but the newProjectRequestsStruct as a name of a struct is an interesting choice.